admin/src/store/store.js

```diff
@@ -35,7 +35,7 @@ const store = new Vuex.Store({
   ],
   state: {
     token: CONFIG.DEBUG_DISABLE_AUTH ? 'validToken' : null,
-    moderator: 'Dertest Moderator',
+    moderator: { name: 'Dertest Moderator', id: 0 },
```

Comment:
consider to use `verifyLogin` from the UserResolver. It just requires a token and will return a user object of the logged in user

Reply:
Good Catch!

Will add this to the list of todos!

Reply:
Will fix this in branch ``adminPendingCreationQueries`` since we just need it their.